README.md

```diff
@@ -1,2 +1,23 @@
 # ff-scala
 A Scala implementation of the Flippy Flop data structure.
+
+## Background
+
+A Flippy Flop is an unpredictable data structure.
+Flip (insertion) and flop (removal) operations succeed at random.
+
+## Usage
+
+As you'll see in the example below, the Flippy Flop is sufficiently unpredictable.
+
+### Scala
+
+Scala Code Here?
```

Comment:
Thanks @Mosuswalks, any chance you want to take a crack at adding the scala example?

Reply:
Hey @shakeelmohamed, I would love to give it a try!
I'm not familiar with Scala but it seems to be somewhat similar to Java.

Do you have the Scala Class i can reference?

Reply:
@Mosuswalks This is the only program I've ever written in Scala so you're not far behind 😄 
The code is on the develop branch: https://github.com/flippy-flop/ff-scala/blob/develop/FlippyFlop.scala